tests/test_s3blobstore.py

```diff
@@ -135,5 +135,21 @@ def test_find_next_missing_parts_multiple_requests(self):
             else:
                 self.assertIn(ix, res)
 
+    def test_check_bucket_exists(self):
+        """
+        Ensure that the ``check_bucket_exists`` method returns true for FIXTURE AND TEST buckets.
+        """
+        handle = self.handle  # type: BlobStore
+        self.assertEqual(handle.check_bucket_exists(self.test_fixtures_bucket), True)
+        self.assertEqual(handle.check_bucket_exists(self.test_bucket), True)
+
+    def test_get_bucket_region(self):
+        """
+        Ensure that the ``get_bucket_region`` method returns true for FIXTURE and TEST buckets.
+        """
+        handle = self.handle  # type: BlobStore
+        self.assertEqual(handle.get_bucket_region(self.test_fixtures_bucket), 'us-east-1')
```

Comment:
There needs to be a non-us-east-1 test as well.

Reply:
great idea, do you have any recommendations as far as buckets available in the context of the test?